server/Cargo.toml

```diff
@@ -105,6 +105,7 @@ hashlru = { version = "0.11.0", features = ["serde"] }
 path-clean = "1.0.1"
 prost = "0.12.3"
 prometheus-parse = "0.2.5"
+sha2 = "0.10.8"
```

Comment:
why not use [ulid](https://github.com/ulid/spec) that we use for deployment ids?

Reply:
I am hashing the timestamp, so I needed sha2